mm2src/coins/lightning.rs

```diff
@@ -280,7 +286,31 @@ impl MarketCoinOps for LightningCoin {
     fn base_coin_balance(&self) -> BalanceFut<BigDecimal> { unimplemented!() }
 
     fn send_raw_tx(&self, _tx: &str) -> Box<dyn Future<Item = String, Error = String> + Send> { unimplemented!() }
-
+    fn get_raw_tx(&self, tx: &str) -> Box<dyn Future<Item = String, Error = MmError<GetRawTransactionError>> + Send> {
+        match H256Json::from_str(tx) {
```

Comment:
We have the `utxo_common` module that we use for fns implementing the same functionality for UTXO coins. Could you please extract this fn there and use `utxo_common::get_raw_tx` here instead?